dist/rpm/openQA.spec

```
@@ -314,6 +314,13 @@ Use this package to install munin scripts that allow to monitor some openQA
 statistics.
 %endif
 
+%package openqa-cli-bash-completion.bash
+Summary:        Bash Completion for %{name}
+Requires:       bash-completion
+Supplements:    (%{name} and bash)
```

I'm not completely sure, but this should probably be:
```suggestion
%package bash-completion
Summary:        Bash Completion for %{name}
Requires:       bash-completion
Supplements:    (openQA-client and bash)
```

---

`%{name}` is "openQA" and for subpackage, the name is automatically prepended so we will end up with openQA-bash-completion package which maybe not what we want so we might want to be even more crazy

```suggestion
%package client-bash-completion
Summary:        Bash Completion for %{name}-client
Requires:       bash-completion
Supplements:    (%{name}-client and bash)
```